ofrak_core/ofrak/core/magic.py

```diff
@@ -22,18 +21,47 @@ class Magic(ResourceAttributes):
     descriptor: str
 
 
+class _LibmagicDependency(ComponentExternalTool):
+    try:
+        import magic
+    except ImportError:
+        magic = None
+
+    def __init__(self):
+        super().__init__(
+            "libmagic",
+            "https://linux.die.net/man/3/libmagic",
+            install_check_arg="",
+            brew_package="libmagic",
```

Comment:
Should this also have `apt_package="libmagic1"`?

Reply:
I think the proper URL is https://www.darwinsys.com/file/. And yes, `apt_package="libmagic1"`